_data/live_deployments.json

```diff
@@ -1594,6 +1594,26 @@
           "highlight": "1"
         }
       ]
+    },
+    {
+      "name": "EMBL-EBI Training Courses",
+      "description": "To enable EMBL to deliver world-leading training in bioinformatics and scientific service provision to the research community, empowering scientists at all career stages to make the most of biological data, and strengthening bioinformatics capacity across the globe.",
+      "url": "https://www.ebi.ac.uk/training",
+      "nodes": "EBI",
+      "profiles":[
+        {
+          "profileName": "Course",
+          "conformsTo": "1.0-RELEASE",
```

Comment:
@AlasdairGray quick question. 1.0 RELEASE is coming but not available yet on the website. Is it ok to have live deploys referencing the release rather than the draft?

Reply:
Same discussion at https://github.com/BioSchemas/bioschemas.github.io/pull/560#discussion_r930835863